recipe/conda_build_config.yaml

```
@@ -1,2 +1,6 @@
+MACOSX_SDK_VERSION:
+  - '11.0'
 vc:
-  - 14
\ No newline at end of file
+  - 14
+qt6_main:
+  - '6.6'
```

I think you'll need qt 6.7; pyside6 hasn't been building for 6.6 since over half a year, which is where the conflicts are coming from AFAICT
```suggestion
qt6_main:
  - '6.7'
```